bin/out

```diff
@@ -20,15 +20,25 @@ elif [ ! -f "$SRC/$TEMPLATE" ]; then
   exit 1
 fi
 
+
 ARGS=""
-VAR_FILE=$(jq -r '.params.var_file // empty' /tmp/input)
-if [ ! -f "$SRC/$VAR_FILE" ]; then
-  echo "var_file $SRC/$VAR_FILE does not exist" >&2
-  exit 1
+VAR_FILE=$(jq '.params.var_file // empty' /tmp/input)
+
+if [ $(echo $VAR_FILE | jq 'if type=="array" then true else false end') = "true" ]; then
+  FILES=$(echo $VAR_FILE | jq -r '.[]')
 else
-  ARGS="-var-file=$SRC/$VAR_FILE"
+  FILES=$(echo $VAR_FILE | jq -r '.')
 fi
```

Comment:
I think these two blocks could be condensed to a single `jq` command… something like:

```
  FILES=$(jq -r '.params.var_file // empty | if type =="array" then .[] else . end' /tmp/input)
```